.changeset/spotty-nails-jam.md

```diff
@@ -0,0 +1,15 @@
+---
+"skuba": patch
+---
+
+template/*: Update to Node 20
+
+Consider upgrading the Node.js version for your project across:
+
+- `.nvmrc`
+- `package.json#/engines/node`
+- `serverless.yml`
+- `@types/node` package version
+- CI/CD configuration (`.buildkite/pipeline.yml`, `Dockerfile`, etc.)
+
+If you are updating your AWS Lambda runtime to `nodejs20.x`, consider reading the [release announcement](https://aws.amazon.com/blogs/compute/node-js-20-x-runtime-now-available-in-aws-lambda/) as there are some breaking changes with this upgrade.
```

Comment:
> For optimal performance, and to have full control over your code dependencies, we recommend [bundling and minifying the AWS SDK](https://aws.amazon.com/blogs/developer/reduce-lambda-cold-start-times-migrate-to-aws-sdk-for-javascript-v3/) in your deployment package, rather than using the SDK included in the runtime. For more information, see [Optimizing Node.js dependencies in AWS Lambda](https://aws.amazon.com/blogs/compute/optimizing-node-js-dependencies-in-aws-lambda/).

hmmm